eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -30,6 +31,10 @@ parameters:
                 template: %ezplatform.default_view_templates.content.line%
                 match: []
         embed:
+            image:
+                template: '%ezplatform.default_view_templates.content.embed_image%'
+                match:
+                    IsEmbedImage: true
```

Comment:
It comes from an interpretation of this value in the Embed RichText renderer. When this class is in the XML, it is mapped to an `isImage` objectParameter. The `class` is added by PlatformUI in order to distinguish normal embeds from image embeds.

But I suppose to matching on the content type identifier is more flexible. I'll revert the changes then.